src/tabular-api-surface.ts

```diff
@@ -736,12 +759,30 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
         err = null;
       }
 
+      const mutationErrors = Array.from(mutationErrorsByEntryIndex.values());
       if (mutationErrorsByEntryIndex.size !== 0) {
-        const mutationErrors = Array.from(mutationErrorsByEntryIndex.values());
         callback(new PartialFailureError(mutationErrors, err));
         return;
       }
-
+      if (err && timeoutExceeded) {
+        callback(
+          new PartialFailureError(
+            /* We concatenate an error onto the list for each pending entry
+              because it is required in TestMutateRows_Generic_DeadlineExceeded.
```

Comment:
Let's update this comment to describe what the expected client behavior is instead.  Something like:
"If there's an RPC level failure and the mutation entries don't have a status code, the RPC level failure error code will be used as the entry failure code."

Reply:
Yup. Updated the comment.